gateway/single-cluster/global-l7-xlb-https-backend/README.md

```diff
@@ -0,0 +1,551 @@
+    ```
+
+1. Validate the HTTP route. The output should look similar to this.
+
+    ```bash
+    $ kubectl describe httproute foo -n gxlb-demo-ns1
+    Name:         foo
+    Namespace:    gxlb-demo-ns1
+    Labels:       <none>
+    Annotations:  <none>
+    API Version:  networking.x-k8s.io/v1alpha1
+    Kind:         HTTPRoute
+    Metadata:
+      Creation Timestamp:  2022-01-11T08:25:42Z
+      Generation:          1
+      Managed Fields:
+        API Version:  networking.x-k8s.io/v1alpha1
+        Fields Type:  FieldsV1
+        fieldsV1:
+          f:metadata:
+            f:annotations:
+              .:
+              f:kubectl.kubernetes.io/last-applied-configuration:
+          f:spec:
+            .:
+            f:gateways:
+              .:
+              f:allow:
+              f:gatewayRefs:
+            f:hostnames:
+            f:rules:
+        Manager:      kubectl-client-side-apply
+        Operation:    Update
+        Time:         2022-01-11T08:25:42Z
+        API Version:  networking.x-k8s.io/v1alpha1
+        Fields Type:  FieldsV1
+        fieldsV1:
+          f:status:
+            .:
+            f:gateways:
+        Manager:         GoogleGKEGatewayController
+        Operation:       Update
+        Subresource:     status
+        Time:            2022-01-11T08:26:23Z
+      Resource Version:  18021
+      UID:               783b0d34-7cc9-4a9e-b604-7655a781359a
+    Spec:
+      Gateways:
+        Allow:  FromList
+        Gateway Refs:
+          Name:       external-http
+          Namespace:  default
+      Hostnames:
+        foo.$DOMAIN
+      Rules:
+        Forward To:
+          Port:          8443
+          Service Name:  foo
+          Weight:        1
+        Matches:
+          Path:
+            Type:   Prefix
+            Value:  /
+    Status:
+      Gateways:
+        Conditions:
+          Last Transition Time:  2022-01-11T08:42:03Z
+          Message:
+          Reason:                RouteAdmitted
+          Status:                True
+          Type:                  Admitted
+          Last Transition Time:  2022-01-11T08:42:03Z
+          Message:
+          Reason:                ReconciliationSucceeded
+          Status:                True
+          Type:                  Reconciled
+        Gateway Ref:
+          Name:       external-http
+          Namespace:  default
+    Events:
+      Type    Reason  Age                From                   Message
+      ----    ------  ----               ----                   -------
+      Normal  ADD     17m                sc-gateway-controller  gxlb-demo-ns1/foo
+      Normal  SYNC    72s (x5 over 16m)  sc-gateway-controller  Bind of HTTPRoute "gxlb-demo-ns1/foo" to Gateway "default/external-http" was a success
+      Normal  SYNC    72s (x5 over 16m)  sc-gateway-controller  Reconciliation of HTTPRoute "gxlb-demo-ns1/foo" bound to Gateway "default/external-http" was a success
+    ```
+
```

Comment:
I'd add a step here that reminds to add the new ip to foo.DOMAIN and bar.DOMAIN in the dns config